tasks/main.yml

```diff
@@ -132,7 +132,11 @@
   become: "no"
 
 - name: "Generate special sampledata transfers"
-  command: "~/archivematica-sampledata/createtransfers.py create-variously-encoded-files"
+  command: "~/archivematica-sampledata/createtransfers/createtransfers.py {{ item }}"
```

Comment:
@jrwdunham @ross-spencer ^ it'd be nice to have a Makefile in `archivematica-sampledata` so the consumer doesn't need to know any of these details.

Reply:
I've just remembered that we already have an issue for this!
https://github.com/artefactual/archivematica-sampledata/issues/24